tderenko/ithillel_laravel
# [homework-27] User Authentication

issue

1. Доробити сіди
2. Встановити Laravel UI або Laravel Breeze
3. Зробити реєстрацію користувача. Авторизація по ідеї має і так працювати (але перевірте)

# Test Plan
1. run npm build
2. go to `http://localhost/register`
3. fill all fields and submit
![Screenshot (2)](https://user-images.githubusercontent.com/18249681/193645149-137ce161-f33b-4a57-a8e6-18e80a6d83c5.png)
4. you should be logined 
![Screenshot (3)](https://user-images.githubusercontent.com/18249681/193645257-ce29a21e-98f8-4cbb-88de-77f1619709b7.png)
